cudaaligner/src/myers_gpu.cu

```diff
@@ -28,27 +28,51 @@ namespace claragenomics
 namespace cudaaligner
 {
 
+constexpr int32_t warp_size = 32;
```

Comment:
`CGA_CONSTEXPR` from here https://github.com/clara-genomics/ClaraGenomicsAnalysis/blob/89c411633de49dcdafe34492259d1d14722fe808/common/config/include/claragenomics/cga_config.hpp#L27 for compatibility with old CUDA versions

Reply:
`constexpr` variables should be ok. The problem are `constexpr` functions. The CI CUDA 9.0 passed, too.

Reply:
Ah I see. Yes missed that the tests all passed